docs_rtd/feature_annotation_parser.py

```diff
@@ -34,7 +34,9 @@
     "expectation_completeness",
 ]
 
-AnnotatedNode = namedtuple("AnnotatedNode", ["name", "path", "annotation", "type_"])
+AnnotatedNode = namedtuple(  # noqa: PYI024
+    "AnnotatedNode", ["name", "path", "annotation", "type_"]
+)
```

Comment:
@Shinnnyshinshin this class doesn't appear to be used anywhere in our codebase.
Can I delete it or should this be used somewhere?

Reply:
I think this should be good to delete :)